.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ workflows:
     jobs:
       - linux-build
       - linux-build-options
-      - linux-parquet-s3-build
```

Comment:
@majetideepak Is linux-parquet-hive-storage-adapters-build replacing remove linux-parquet-s3? Why remove it?

Reply:
@shiyu-bytedance this is simply a rename needed to accommodate hdfs testing.